src/main/java/com/photolocos/enterprise/dao/IPhotoDAO.java

```diff
@@ -27,7 +27,7 @@
      * @param tags A array of list of keywords, used to return specific photos
      * @return A Set of photo objects if photos with similar tags are found, otherwise null
      */
-    Set<PhotoDTO> fetchByTag(List<String> tags);
+    Set<PhotoDTO> fetchByTag(String[] tags);
```

Comment:
Should we see about making this a list of comma separated words instead of an array?

Reply:
Yeah...not sure which would be best